### PR TITLE
Add keyboard and right-click shortcuts 

### LINKED ISSUE #35: Add shortcuts
Make pressing Enter on keyboard equivalent of pressing the OK button. It's especially useful when creating a new production sheet. The form already closes on pressing Esc.

Optionally, make Ctrl+T equivalent to pressing "+" on top left. It's analogous to making a new tab in Firefox and Chrome.

### PR DESCRIPTION
**Pull request by DaleStan**
This closes #35 by adding the Enter/Return, right-click, and Ctrl+T behaviors mentioned there. It also adds right-click on products and ingredients to remove or create ordinary links (by asking "What can we clear when right-clicking on an ingredient?"), and right-click on desired products to remove and unlink them.

I'll admit I'm not a fan of 5676ce7a, but I was unable to figure out how to get a single InputSystem to send Enter to the correct window when multiple OS windows were open.

### CODE REVIEW

**Comment by have-fun-was-taken** [comment]
This is awesome. Thank you for implementing this! I will check it out in 10 hours.

**Comment by have-fun-was-taken** [comment]
Checked the basic functionality of Enter, Ctrl+T, and Right Mouse Button -- things work as expected.

File: Yafc.UI/Core/Ui.cs [hunk 1]
```diff
@@ -55,7 +66,7 @@ public static class Ui {
 
         public static void ProcessEvents() {
             try {
-                var inputSystem = InputSystem.Instance;
+                InputSystem? inputSystem = ActiveInputSystem;
```
[the remark this answers is not in the document]
Comment (DaleStan) [reply Yafc.UI/Core/Ui.cs:69]: That seems to be fine. I was initially worried about the `SDL.SDL_EventType.SDL_WINDOWEVENT` case, but I never ran into a null `inputSystem` while I was testing.

**Review by have-fun-was-taken — APPROVED** [review]
Waiting for @veger to resolve the conversation.

**Review by veger — APPROVED** [review]

**Pull request merged**